k8s/helmfile/env/local/elasticsearch.values.yaml.gotmpl

```diff
@@ -1,3 +1,6 @@
+labels:
+  sidecar.istio.io/inject: "true"
```

Comment:
This means that when testing this locally, you will need to delete the `elasticsearch-master-0|1|2` pods manually in order to trigger the proxy injection. Alternatively, you can remove the label and re-apply, which will then trigger proxy creation as the webhook is listening.